src/ovs/aca_ovs_control.cpp

```diff
@@ -56,11 +56,13 @@ int ACA_OVS_Control::control()
 {
   int overall_rc = EXIT_SUCCESS;
 
-  char target[g_ofctl_target.size() + 1];
+  // char target[g_ofctl_target.size() + 1];
+  char *target = new char[g_ofctl_target.size() + 1];
```

Comment:
interesting to see compiler flagged this as warning. keep in mind you will need to free the newly allocated memory before function exit.

@cj-chung - please review this to make sure the change is correct

Reply:
If using `new` to create a `char*`, should we explicitly to `delete[] target`?